includes/utils.inc

```diff
@@ -54,12 +54,12 @@ function wf_crm_field_options($field, $context, $data) {
     elseif ($name == 'privacy') {
       $ret = wf_crm_get_privacy_options();
     }
-    elseif ($table === 'other') {
-      if ($field['table'] === 'tag') {
+    elseif ($table === 'other' || ($table === 'case' && strpos($name, 'tag_') === 0)) {
```

Comment:
This disallows us from getting activity tags.
Could you please take my previous suggestion and remove the outer `elseif`.

Reply:
Updated as suggested.